### PR TITLE
chore(a11y): add documentation article for the new default ocean blue…

### PR DESCRIPTION
**Pull request by xristianstefanov**

### CODE REVIEW

File: accessibility/accessibility-swatch.md [hunk 1]
```diff
@@ -14,9 +14,17 @@ The [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/) sectio
 
 You can test the accessibility swatch in our [live demos](https://demos.telerik.com/blazor-ui/grid/overview) - it is initially selected in the `Change Theme` menu.
 
-Telerik UI for Blazor distributes the Default Ocean Blue theme swatch via several resources. Read more in the [Swatch Distribution]({%slug themes-swatch-distribution%}) article.
+# Default Ocean Blue Accessibility Swatch
 
-In addition, it is also available in:
+The Default Ocean Blue Accessibility Swatch is built on top of [Ocean Blue](#ocean-blue-accessibility-swatch). The difference is that it has contrast focus indicators enabled and enforces compliance with WCAG requirements.
```
Comment (dimodi) [inline accessibility/accessibility-swatch.md:19]:
```suggestion
The Default Ocean Blue Accessibility Swatch is built on top of [Ocean Blue](#ocean-blue-accessibility-swatch). The difference is that it has contrast focus indicators and is compliant with WCAG requirements.
```

File: accessibility/accessibility-swatch.md [hunk 2]
```diff
@@ -14,9 +14,17 @@ The [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/) sectio
...
+
+You can test the accessibility swatch in our [live demos](https://demos.telerik.com/blazor-ui/grid/overview) - select the `Default Ocean Blue` option in the `Change Theme` menu.
```
Comment (dimodi) [inline accessibility/accessibility-swatch.md:21]:
```suggestion
You can test the accessibility swatch in our [live demos](https://demos.telerik.com/blazor-ui/grid/overview) - select the `Default Ocean Blue` option in the `Change Theme` dropdown menu.
```

File: accessibility/accessibility-swatch.md [hunk 3]
```diff
@@ -14,9 +14,17 @@ The [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/) sectio
...
+
+> **Limitations:** ColorPalette fails WCAG success criteria 1.4.11. Non-text contrast for the focus indicator on its items.
```
Comment (dimodi) [inline accessibility/accessibility-swatch.md:23]:
```suggestion
> **Limitations:** ColorPalette fails WCAG success criteria 1.4.11. - "Non-text contrast for the focus indicator on its items".
```

**Review by dimodi — APPROVED** [review]

File: accessibility/accessibility-swatch.md (hunk 1 above)
Comment (xristianstefanov) [reply accessibility/accessibility-swatch.md]: Done

File: accessibility/accessibility-swatch.md (hunk 3 above)
Comment (xristianstefanov) [reply accessibility/accessibility-swatch.md]: Done

**Pull request merged**